lib/ssl/doc/src/using_ssl.xml

```diff
@@ -247,7 +247,64 @@ ok</code>
    #{cipher => aes_256_cbc,key_exchange => ecdhe_ecdsa,
      mac => sha384,prf => sha384}, ...]
     </code>
-  </section>      
+  </section>
+
+    <section>
+    <title>Customizing signature algorithms(TLS-1.2)/schemes(TLS-1.3)</title>
+
+    <p> Starting from TLS-1.2 signature algorithms (called signature
+    schemes in TLS-1.3) is something that can be negotiated and hence
+    also configured. These algorithms/schemes will be used for digital
+    signatures in protocol messages and in certificates. </p>
+
+    <note><p>TLS-1.3 schemes have atom names whereas TLS-1.2
+    configuration is two tuples composed by one hash algorithm and one
+    signature algorithm. When both versions are supported the
+    configuration can be a mix of these as both versions might be
+    negotiated. All <c>rsa_pss</c> based schemes are back ported to
+    TLS-1.2 and can be used aslo in a TLS-1.2 configuration. Also note
```

Comment:
Not an expert on grammar, but shouldn't you put a coma (`,`) after "Also note" ?